tests/ops/qubit/test_parametric_ops.py

```diff
@@ -139,6 +140,17 @@ def multi_dot_broadcasted(matrices):
     return reduce(dot_broadcasted, matrices)
 
 
+class TestSparseOperators:
+    @pytest.mark.parametrize("op", SPARSE_OPERATIONS)
+    def test_validity(self, op):
+        """Test that the operations are valid."""
+        if isinstance(op, qml.GlobalPhase):
+            pytest.skip("GlobalPhase has bugs related with sparse matrices")
```

Comment:
If it's a bug, use `xfail` instead of `skip`, and add the associated sc story.

Reply:
Thanks for catching this. it is not supposed to be skipped here; this line should be from some misalignment between my branches